scanner/src/funTest/kotlin/storages/AbstractStorageFunTest.kt

```diff
@@ -313,7 +313,7 @@ abstract class AbstractStorageFunTest : WordSpec() {
             }
 
             "find a scan result if the revision was detected from a version" {
-                val scanResult = ScanResult(provenanceWithOriginalVcsInfo, scannerDetails1, scanSummaryWithFiles)
+                val scanResult = ScanResult(provenanceWithoutRevision, scannerDetails1, scanSummaryWithFiles)
```

Comment:
While at it, should we maybe change "detected" in the test names to "resolved" here and below?

Reply:
Good idea!